private-api/imcore-documentation.md

```diff
@@ -577,9 +584,10 @@ IMChat *chat = [BlueBubblesHelper getChat: data[@"chatGuid"] :transaction];
     NSMutableAttributedString *bcString = [[NSMutableAttributedString alloc] initWithString: data[@"backwardsCompatibilityMessage"]];
     NSInteger index = data["@partIndex"];
     // send the edit
-    [chat editMessage:(message) atPartIndex:(index) withNewPartText:(editedString) backwardCompatabilityText:(bcString)];
+    [chat editMessage:(message) atPartIndex:(index) withNewPartText:(editedString) backwardCompatibilityText:(bcString)];
```

Comment:
This should not be corrected, its actually spelled incorrectly on the macOS headers, funnily enough.

Reply:
Fixed